src/pkg/egress/syslog/https_batch.go

```diff
@@ -0,0 +1,116 @@
+package syslog
+
+import (
+	"bytes"
+	"crypto/tls"
+	"time"
+
+	"code.cloudfoundry.org/go-loggregator/v9/rpc/loggregator_v2"
+	metrics "code.cloudfoundry.org/go-metric-registry"
+	"code.cloudfoundry.org/loggregator-agent-release/src/pkg/egress"
+)
+
+const BATCHSIZE = 256 * 1024
+
+type HTTPSBatchWriter struct {
+	HTTPSWriter
+	msgBatch     bytes.Buffer
+	batchSize    int
+	sendInterval time.Duration
+	sendTimer    *TriggerTimer
+	egrMsgCount  float64
+}
+
+func NewHTTPSBatchWriter(
```

Comment:
Can you please add tests for this writer.

Reply:
Some tests are already added, do we need tests for the things already done by the httpsWriter (Error handling and the likes, which is anyways the same?)